CONTRIBUTOR-LADDER.md

```diff
@@ -0,0 +1,129 @@
+
+### Community Contributor
+
+* Responsibilities:
+    * Must follow the [Cilium CoC](https://github.com/cilium/cilium/blob/master/CODE_OF_CONDUCT.md)
+    * Follow the [contributing guide](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/) 
+
+* Qualifications:
+Anybody that is participating in the Cilium community is welcome. Any contribution counts. Some possible forms of contribution are:
+    * Participating in community discussions
+    * Helping other users in Slack or in person
+    * Submitting bug reports
+    * Trying out new releases
+    * Attending community events
+    * Giving talks about Cilium virtually or in person
+    * Report and sometimes resolve issues
+    * Occasionally submit PRs
+    * Writing a CFP
+    * Contribute to the documentation
+    * Show up at community meetings and/or leads and/or takes notes
+    * Answer questions from other community members
+    * Submit feedback and comments on issues and PRs
+    * Test releases and patches and submit reviews
+    * Promote the project in public
+
+Non-code contributions can be documented by making a PR to add yourself to CONTRIBUTORS.md. 
+
+* Privileges:
+    * Ability to be assigned to Issues and ask for Reviews
+ 
+Becoming a Community Contributor is the first step towards becoming an Organization Member
+
+### Organization Member
+Description: An Organization Member is an established contributor who regularly participates in the project. Organization Members have privileges in project repositories and as such are expected to act in the interests of the whole project. Organization Members can take on additional roles within the project and can work more independently, for example triggering CI. 
+
+An Organization Member has all the rights and responsibilities of a Community Contributor, plus:
+
+* Responsibilities:
+    * Continues to contribute regularly
+    * Work to accomplish the tasks they volunteer to do within the project
+
+* Qualifications:
+    * Must have successful contributions to the project, including at least one of the following:
+        * Written and/or reviewed PRs,
+        * Reviewed PRs,
+        * Made contributions that resolved Issues,
+        * Or some equivalent combination or contribution
+    * Must be actively contributing to at least one project area, where the privileges of Organization Member will be beneficial
+
+* Privileges:
+    * Can trigger CI
+    * Can leave reviews on PRs
+    * Can recommend other contributors to become Cilium Github Org Members
+    * Can nominate themselves to other roles within Cilium
+
+The process for a Contributor to become an Organization Member is as follows:
+
+1. Open a PR in the member management file
+2. Receive two “LGTM” comments from any Org Members, and approval by a committer who merges the PR
```

Comment:
Nit: I'd avoid missing straight and curly quotes in the doc. Well, I'd avoid curly quotes altogether.